adr/0023-multi-node-kubernetes.md

```diff
@@ -0,0 +1,75 @@
+# 22. Multi-Node Kubernetes with `zarf init`
+
+Date: 2024-01-23
+
+## Status
+
+Pending
+
+## Context
+
+Currently today, when allowing `zarf init` to handle cluster creation, Zarf doesn't have ability to automatically or semi-automatically provision itself across multiple nodes.  The idea here would be to allow for horizontal scalability across multiple virtual or physical nodes for site reliability and automatic failover.
+
+The first pass would consider scaling horizontally with the Embedded DB model.  There would be minimal changes on the current k3s.service config.  The change required here would be to include a shared token.  By default, if K3S doesn't receive a token it will auto generate one.  You can reset this on an existing cluster by running `k3s token rotate --new-token=foo`.
```

Comment:
I recommend being more explicit about which datastore option this uses and linking to it

```suggestion
The first pass would consider scaling horizontally with the [High Availability Embedded etcd](https://docs.k3s.io/datastore/ha-embedded) model.  There would be minimal changes on the current k3s.service config.  The change required here would be to include a shared token.  By default, if K3S doesn't receive a token it will auto generate one.  You can reset this on an existing cluster by running `k3s token rotate --new-token=foo`.
```